.travis.yml

```diff
@@ -1,8 +1,10 @@
 language: ruby
 sudo: false
 rvm:
-  - 2.2.4
```

Comment:
Why ruby 2.2 support was removed? This gem still supports it.

Reply:
Sorry, will readd, was just quickly fixing CI.